### PR TITLE
PDAL is now managed in the OSGeo4Mac Tap

### PR DESCRIPTION
**Pull request by hobu**
This PDAL recipe is very old, and a full PDAL build depends upon a number of recipes in the OSGeo4Mac tap. This recipe is now managed there.

https://github.com/OSGeo/homebrew-osgeo4mac

### CODE REVIEW

**Comment by dunn** [comment]
If we do this it will also need to be marked in the list of tap migrations: https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/tap_migrations.rb

File: Library/Homebrew/tap_migrations.rb [hunk 1]
```diff
@@ -155,6 +155,7 @@
   "pari" => "homebrew/x11",
   "pathfinder" => "homebrew/boneyard",
   "pcb" => "homebrew/x11",
+  "pdal" => "homebrew/osgeo",
```
Comment (xu-cheng) [inline Library/Homebrew/tap_migrations.rb]: This is incorrect. It should be `OSGeo/osgeo4mac`

**Comment by mikemcquaid** [comment]
I'd rather we didn't migrate formulae to general purpose taps (e.g. the software is not maintained by the creators of the software) outside the Homebrew organisation. I'm always open to taps joining the Homebrew organisation, however.

**Comment by hobu** [comment]
> I'd rather we didn't migrate formulae to general purpose taps (e.g. the software is not maintained by the creators of the software) outside the Homebrew organisation.

Two items on this point. First, OSGeo4Mac is a special purpose tap with a bunch of related open source geo software. The problem with the PDAL recipe in mainline Homebrew is that a bunch of libraries it optionally depends on for functionality are provided by the OSGeo4Mac tap. The mainline recipe is very out of date, and to bring it up to par would require adding a bunch of other libraries that practically no one uses to mainline Homebrew.

Second, I *am* the creator of PDAL, and I have been very happy with Homebrew as the packaging solution for it. I want to give my users the easiest mechanism to get the library with all of its features available, and the OSGeo4Mac tap is probably the best vehicle for that at this time. I have merge rights in that repo, although I defer to @dakarto unless he no longer has any interest.

What would it mean for OSGeo4Mac to "join the Homebrew organization"?

**Comment by dunn** [comment]
> What would it mean for OSGeo4Mac to "join the Homebrew organization"?

We could transfer the repository to the Homebrew organization, so it would be located at Homebrew/homebrew-osgeo4mac (or more likely just Homebrew/homebrew-osgeo). You all would remain the maintainers, but it would be an "official" tap, like homebrew-science.

**Comment by hobu** [comment]
I think it would be a great idea to have an official Homebrew/osgeo. Any thoughts @dakarto?

**Comment by dunn** [comment]
Any further thoughts from the @osgeo team?

**Comment by dakcarto** [comment]
Hi,

Sorry for the belated response. My day job has eaten up all my time recently (like for the last 10 months).

I am still in favor of migrating the OSGeo4Mac tap formulae over to a `homebrew-geospatial` general tap, as previously discussed. I now have more free time and would like to work on this.

I have done some non-standard things in many formulae in the OSGeo4Mac tap and need to clean those up first. I think the tap will still remain for specific tasks/formulae outside of mainstream Homebrew, though.

**Comment by mikemcquaid** [comment]
@dakcarto I don't mind moving it over first and then cleaning it up afterwards!

**Comment by mikemcquaid** [comment]
@dakcarto Have created a @Homebrew/geospatial team and invited you to it with admin access. You should be able to transfer the repo after you accept the invitation and then I'll give you (and anyone else you want) admin access to it.

**Comment by dakcarto** [comment]
@mikemcquaid Thanks! Will work on this ASAP.

**Comment by mikemcquaid** [comment]
@dakcarto Great! Shout if you need anything.

**Comment by dunn** [comment]
Any update on the migration?

**Comment by hobu** [comment]
Let me know any way I might be able to help @dakcarto

**Comment by dunn** [comment]
Ping.

**Comment by hobu** [comment]
FYI PDAL is now having official releases. See http://www.pdal.io/download.html#current-release-s or https://github.com/PDAL/PDAL/releases/tag/1.1.0 for more detail.

**Comment by dunn** [comment]
:bell:

**Comment by apjanke** [comment]
Ping. Who makes the call on what to do here? Legacy-homebrew is deprecated, so need to either close this out as "won't change" or migrate it to the new homebrew/core tap.

**Comment by hobu** [comment]
It doesn't make a lot of difference to me where it lives. I will make PRs for the recipe and any of its supporting libraries to wherever they need to go.

**Comment by apjanke** [comment]
Given that it's lived in core Homebrew for 5+ years now, I think it makes more sense to keep it there or at least push forward with a `homebrew/geospatial` organizational tap.

I'm interested in geospatial software myself. I'll read through the stuff in your tap to get familiarized with it and see what might be needed to get this moved in to @Homebrew/geospatial or aligned with core Homebrew standards.

**Comment by dakcarto** [comment]
Hi,

I am preparing formulae in the [OSGeo4Mac tap](https://github.com/OSGeo/homebrew-osgeo4mac) for migration to @Homebrew/geospatial. The `pdal` formulae is an excellent first choice. The formulae in the OSGeo4Mac tap does need updated however.

**Comment by MikeMcQuaid** [comment]
Passing on this in favour of @dakcarto's migration.

**Pull request closed without merging**